### ISSUE TITLE
Use string interpolation instead of manual concatenation

### ISSUE THREAD

**Issue by sole** [mozilla/fireplay-sublime#23]
Python has fantastic string interpolation equivalent to C's `printf`. Instead of doing

``` python
"Error: Path '" + target_app_path + "' is neither a directory or a .zip file to represent the location of a FFOS packaged app!"
```

you could be doing

``` python
"Error: Path '%s' is neither a directory or a .zip file to represent the location of a FFOS packaged app!" % (target_app_path)
```

that makes it so much easier to add translations as discussed in #20

**Comment by nicola** [comment]
Ha! Here is where the problem comes with Python 2.6 (used in Sublime 2), I will explore on that.

**Comment by nicola** [comment]
Fixed 6f39f6d632c2d4e1f439a38228b756c808e65796
